rogue-thi-app/package.json

```diff
@@ -30,6 +30,7 @@
     "ical-generator": "^3.6.1",
     "idb": "^7.0.0",
     "leaflet": "^1.7.1",
+    "moment": "^2.29.4",
```

Comment:
moment [is officially a legacy project](https://momentjs.com/docs/#/-project-status/) and should not be used in new code.

Reply:
We could instead use either [Luxon](https://moment.github.io/luxon/) (the direct successor to Moment.js) or [Day.js](https://day.js.org/) (which is more lightweight). Both are quite similar in terms of their API.